pkg/proxy/proxy.go

```diff
@@ -63,19 +64,28 @@ func (p *Proxy) StartProxy() *http.Server {
 	// start server
 	mux := http.NewServeMux()
 	mux.HandleFunc("/", p.handleRequestAndRedirect)
+	mux.HandleFunc("/health", p.health)
```

Comment:
There's no overlapping with the kube health endpoints, right? I guess all kube endpoints start with /api ?

Reply:
Good point, I didn't think about that. It sounds like there is no overlap: https://kubernetes.io/docs/reference/using-api/#api-groups

Reply:
I think that @alexeykazakov didn't mean the k8s api endpoint, but any of the "health" endpoints (there are more of them in k8s https://kubernetes.io/docs/reference/using-api/health-checks/) that k8s cluster provides. If I wanted to check whether my particular member cluster is healthy or not and I wanted to use proxy for it, then I would send a request to the  `/healthz` endpoint.
Looking at the set of the paths that k8s provides, there should be no overlapping. But it could be maybe more transparent if we used for example `/proxyhealth`  endpoint. WDYT?

Reply:
I just read the description of the PR :man_facepalming: yeah, simple `/health` is probably better one :-)

Reply:
Yeah I was thinking about something like /proxyhealth too. Just to make sure it's unique. /health should be ok because it seems there is no overlapping with the kube health endpoints. At least for now. Anyway I leave it to you Rajiv. It's your call :)

Reply:
I changed it to `/proxyhealth` since it's less ambiguous and reduces any possibility of overlap in the future. 🙂